locales/messages.de.json

```
@@ -51,23 +51,23 @@
   "SeeURL": "Weitere Informationen finden Sie unter {url}.",
   "SuggestNewVersionScheme": "Verwenden Sie das Versionsschema „{new_scheme}“ anstelle von „{old_scheme}“ in Port „{package_name}“. \nVerwenden Sie „--{option}“, um diese Überprüfung zu deaktivieren.",
-  "UnsupportedSystemName": "Fehler: VCPKG_CMAKE_SYSTEM_NAME „{system_name}“ konnte keiner vcvarsall-Plattform zugeordnet werden. Unterstützte Systemnamen sind „,“, „Windows“ und „WindowsStore“.",
-  "UnsupportedToolchain": "Fehler: Im Tripel {triplet}: Es wurde keine gültige Toolkettenkombination gefunden.\n    Die angeforderte Zielarchitektur war {arch}\n    Die ausgewählte Visual Studio-Instanz befindet sich unter {path}\n    Die verfügbaren Toolkettenkombinationen sind {list}\n",
+  "UnsupportedSystemName": "Fehler: VCPKG_CMAKE_SYSTEM_NAME „{system_name}“ konnte keiner vcvarsall-Plattform zugeordnet werden. Unterstützte Systemnamen sind „“, „Windows“ und „WindowsStore“.",
+  "UnsupportedToolchain": "Fehler: Im Triplett {triplet}: Es wurde keine gültige Toolset-Kombination gefunden.\n    Die angeforderte Zielarchitektur war {arch}\n    Die ausgewählte Visual Studio-Instanz befindet sich unter {path}\n    Die verfügbaren Toolset-Kombinationen sind {list}\n",
   "UsingManifestAt": "Die Manifestdatei unter {path} wird verwendet.",
   "VSExaminedInstances": "Die folgenden Visual Studio-Instanzen wurden berücksichtigt:",
   "VSExaminedPaths": "Die folgenden Pfade wurden für Visual Studio-Instanzen untersucht:",
   "VSNoInstances": "Es konnte keine vollständige Visual Studio-Instanz gefunden werden.",
-  "VcpkgDisallowedClassicMode": "Fehler: Es wurde kein Manifest (vcpkg.json) oberhalb des aktuellen Arbeitsverzeichnisses gefunden.\nDiese vcpkg-Verteilung verfügt nicht über eine Instanz im klassischen Modus.",
+  "VcpkgDisallowedClassicMode": "Fehler: Es wurde kein Manifest (vcpkg.json) oberhalb des aktuellen Arbeitsverzeichnisses gefunden.\nDiese vcpkg-Verteilung(ROFL) verfügt nicht über eine Instanz im klassischen Modus.",
   "VcpkgHasCrashed": "vcpkg.exe ist abgestürzt.\nBitte senden Sie eine E-Mail an:\n    {email}\nmit einer kurzen Zusammenfassung dessen, was Sie versucht haben, und dem folgende Datenblob:",
   "VcpkgHasCrashedArgument": "{value}|",
   "VcpkgHasCrashedDataBlob": "\nVersion={version}\nEXCEPTION='{error}'\nCMD=",
   "VcpkgInvalidCommand": "Ungültiger Befehl: {command_name}",
   "VcpkgSendMetricsButDisabled": "Warnung: --sendmetrics wurde übergeben, Metriken sind jedoch deaktiviert.",
-  "VersionSpecMismatch": "Fehler: Fehler beim Laden des Ports, weil die Versionsspezifikationen nicht übereinstimmen\n    Pfad: {path}\n    Erwartet: {expected_version}\n    Tatsächlich: {actual_version}",
+  "VersionSpecMismatch": "Fehler: Ports konnten nicht geladen werden, weil die Versionsspezifikationen nicht übereinstimmen\n    Pfad: {path}\n    Erwartet: {expected_version}\n    Tatsächlich: {actual_version}",
```

```suggestion
  "VersionSpecMismatch": "Fehler: Port konnte nicht geladen werden, weil die Versionsspezifikationen nicht übereinstimmen\n    Pfad: {path}\n    Erwartet: {expected_version}\n    Tatsächlich: {actual_version}",
```
Still wondering if it should be plural in the version part of the sentence.

---

(internal only) https://ceapex.visualstudio.com/CEINTL/_workitems/edit/565255